src/vm/class.h

```diff
@@ -1958,11 +1953,11 @@ class EEClass // DO NOT CREATE A NEW EEClass USING NEW!
         VMFLAG_PREFER_ALIGN8                   = 0x00200000, // Would like to have 8-byte alignment
         // unused                              = 0x00400000,
 
-#ifdef FEATURE_COMINTEROP
         VMFLAG_SPARSE_FOR_COMINTEROP           = 0x00800000,
         // interfaces may have a coclass attribute
         VMFLAG_HASCOCLASSATTRIB                = 0x01000000,
```

Comment:
Why did you need to pull in all these? I would think that `enum_flag_ComObject` should have been enough.

Reply:
The methodtablebuilder will do some check for event interface during load time. I can pull  VMFLAG_HASCOCLASSATTRIB  out.

Reply:
All I can see is that these are just used to do some extra validation in methodtablebuilder. I do not think we need to bother with the extra COM-specific validation in methodtablebuilder with `FEATURE_COMINTEROP` disabled.

Reply:
Okay. I am moving these out.